tests/cli/inputs/yaml/complex-types/stdout

```diff
@@ -1,6 +1,11 @@
-test.foo: Foo
-test.bar: Bar
-test.baz: Baz
+test.foo: Foo (default = <could not embed>)
+test.bar: Bar (default = <could not embed>)
+test.baz: Baz (default = <could not embed>)
```

Comment:
To me, this reads like something went wrong. I don't have a clear suggestion, though.

Reply:
`unserializable value`?

Reply:
`<omitted from display>`?